mne/viz/topo.py

```diff
@@ -347,44 +348,38 @@ def _plot_timeseries(ax, ch_idx, tmin, tmax, vmin, vmax, ylim, data, color,
         else:
             ax.plot(times_, data_[ch_idx], color=color_)
 
-    if x_label is not None:
-        ax.set(xlabel=x_label)
-
-    if y_label is not None:
-        if isinstance(y_label, list):
-            ax.set_ylabel(y_label[ch_idx])
-        else:
-            ax.set_ylabel(y_label)
-
     def _format_coord(x, y, labels, ax):
         """Create status string based on cursor coordinates."""
         # find indices for datasets near cursor (if any)
         tdiffs = [np.abs(tvec - x).min() for tvec in times]
         nearby = [k for k, tdiff in enumerate(tdiffs) if
                   tdiff < (tmax - tmin) / 100]
-        timestr = '%6.3f s: ' % x
+        xlabel = ax.get_xlabel()
+        xunit = (xlabel[xlabel.find('(') + 1:xlabel.find(')')]
+                 if '(' in xlabel and ')' in xlabel else 's')
+        timestr = '%6.3f %s: ' % (x, xunit)
```

Comment:
now this will correctly say "Hz" instead of "s" in the statusbar when hovering over a PSD plot.